EvidenceApi/V1/Infrastructure/EvidenceContext.cs

```diff
@@ -17,6 +17,7 @@ public EvidenceContext(DbContextOptions options) : base(options)
         public DbSet<Resident> Residents { get; set; }
         public DbSet<Communication> Communications { get; set; }
         public DbSet<DocumentSubmission> DocumentSubmissions { get; set; }
+        public DbSet<AuditEvent> AuditEvents { get; set; }
```

Comment:
Small thing but in `IEvidenceGateway` you put `CreateAuditEvent` above `CreateEvidenceRequest`. Perhaps we could put `AuditEvents` above `EvidenceRequests`, or move `CreateAuditEvent` to the bottom in `IEvidenceGateway`

Reply:
👍